app/middlewares/albums.js

```diff
@@ -0,0 +1,12 @@
+const errors = require('../errors'),
+  logger = require('../logger');
+
+exports.albumsListAsAdmin = (req, res, next) => {
+  if (req.session.role === 'admin') {
+    return next();
+  }
+  logger.info(req.session.id, req.params.id);
```

Comment:
This log will print random numbers because there is not context. Write something related with this middleware.